synchronizer/models.py

```diff
@@ -370,8 +372,8 @@ class Connector(db.Model):
     name = db.Column(db.String(64), nullable=False)
     server = db.Column(db.String(128), default="")
     login = db.Column(db.String(64), default="")
-    password = db.Column(db.String(120), default="")
-    api_token = db.Column(db.String(120), default="")
+    _password = db.Column(db.String(255), default="")

```

Comment:
As I said earlier, columns in DB should have old names.

Reply:
Is it enough to have 255 symbols for encrypted passwords and API tokens? Is it depend on password/token length?